torch/csrc/api/src/nn/options/activation.cpp

```diff
@@ -29,7 +29,18 @@ SoftminFuncOptions::SoftminFuncOptions(int64_t dim) : dim_(dim) {}
 
 LogSoftmaxFuncOptions::LogSoftmaxFuncOptions(int64_t dim) : dim_(dim) {}
 
-} // namespace functional
+MultiheadAttentionForwardOptions::MultiheadAttentionForwardOptions(
+    int64_t embed_dim_to_check, int64_t num_heads,
+    Tensor in_proj_weight, Tensor in_proj_bias,
+    Tensor bias_k, Tensor bias_v,
+    bool add_zero_attn, double dropout_p,
+    Tensor out_proj_weight, Tensor out_proj_bias
+  ) : embed_dim_to_check_(embed_dim_to_check), num_heads_(num_heads),
```

Comment:
I think you can fix the clang-tidy failures by doing:
``embed_dim_to_check_(std::move(embed_dim_to_check)), num_heads_(std::move(num_heads))``